src/main/java/org/opensha/commons/param/ParameterList.java

```diff
@@ -257,56 +260,51 @@ public <T> Parameter<T> getByIndex(Class<T> type, int index) {
 
 	/** Set's a new value to a Parameter in the list if it exists, else throws exception. */
 	public void setValue(String name, Object value) throws ParameterException, ConstraintException {
-
 		String S = C + ": setValue(): ";
 		if(D) System.out.println(S + "Starting");
 
-		name = getParameterName( name );
+		name = getParameterName(name);
 		int index = getIndexOf(name);
-		if( index !=-1 ) {
+		if (index != -1) {
 			Parameter param = (Parameter)params.get(index);
 			param.setValue(value);
-		}
-		else{
-
+			fireChangeEvent(new ChangeEvent(this));
```

Comment:
this method is for setting a parameter value, not changing the parameters contained in the list. please remove the `fireChangeEvent(...)` call.